.ci/build.yml

```diff
@@ -0,0 +1,43 @@
+parameters:
+  python_versions : []
+
+steps:
+  - ${{ each py_version in parameters.py_versions }}:
+    - task: usePythonVersion@0
+      inputs:
+        versionSpec: '3.6'
+
+    - checkout: self
+      clean: true
+
+    - script: python3 -m pip install --user pyyaml
+      displayName: 'Install PyYaml'
+
+    - script: python3 jellyfin-kodi/.config/generate_xml.py ${{ py_version }}
+      displayName: 'Create ${{ py_version }} addon.xml'
+
+    - task: ArchiveFiles@2
+      displayName: 'Create ${{ py_version }} zip file'
+      inputs:
+        rootFolderOrFile: jellyfin-kodi
+        includeRootFolder: False
+        archiveType: 'zip'
+        tarCompression: 'none'
+        archiveFile: 'plugin.video.jellyfin-${{ py_version }}.zip'
+
+    - task: CopyFilesOverSSH@0
+      displayName: 'Upload to repo server'
+      inputs:
+        sshEndpoint: repository
+        sourceFolder: '${Agent.BuildDirectory}'
+        contents: 'plugin.video.jellyfin-${{ py-version }}.zip'
+        targetFolder: '/srv/repository/incoming/kodi'
+      condition: startsWith(variables['Build.SourceBranch'], 'refs/tags')
```

Comment:
So the pipeline runs on every commit and generate the zip files but it'll only upload the stuff to the server on new tags then?

Reply:
Correct.  I'm thinking about having it attach builds to PRs for ease of testing in the future, but this is a first step and gets auto releases working again

Reply:
It will do that (attach to PRs) in so far that the pipeline artifacts are downloadable from the "checks page" when you open the build.